tests/test_basic_int.py

```diff
@@ -0,0 +1,27 @@
+import h3.api.basic_int as h3
+
+
+def test1():
+    assert h3.geo_to_h3(37.7752702151959, -122.418307270836, 9) == 617700169958293503
```

Comment:
In the future could we write test indexes in hexadecimal? I think we're all used to reading the indexes in that format.

Reply:
In general, that's what I'm doing in the test suite. This test is here to specifically test the `int64` interface, so I think it's reasonable to have it in decimal.

Reply:
Also, just updated the test to use *both*  `617700169958293503` and `0x8928308280fffff` to try and make the equivalence apparent.